client/index.js

```diff
@@ -3,7 +3,10 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import App from './app'
 
-import './style.css'
+import 'normalize.css'
```

Comment:
@AStashevskaya Do we need multiple `import normalize.css` statements? We already have same import in `client/components/Header/header.scss` and `client/components/Order/Order.scss`